nuxt/components/form/Form.vue

```diff
@@ -43,8 +47,11 @@
 <script setup lang="ts">
 import type { BaseValidation } from '@vuelidate/core'
 
+import consola from 'consola'
 import { BackendError } from '~/types/types'
 
+import { useMaevsiStore } from '~/store/index'
+
```

Comment:
1. external imports
1. blank line
1. local imports
in general: type imports after non-type imports
```suggestion
import consola from 'consola'
import type { BaseValidation } from '@vuelidate/core'

import { BackendError } from '~/types/types'
import { useMaevsiStore } from '~/store/index'

```